arduino-ide-extension/src/electron-main/theia/electron-main-application.ts

```diff
@@ -325,12 +327,20 @@ export class ElectronMainApplication extends TheiaElectronMainApplication {
               devTools: true,
               nativeWindowOpen: true,
               openerId: electronWindow?.webContents.id,
+              webSecurity: false,
+              nodeIntegration: false,
             },
           });
           event.newGuest = new BrowserWindow(options);
+
+          const showPlotterWindow = () => {
+            event.newGuest?.show();
+          };
+          ipcMain.on(SHOW_PLOTTER_WINDOW, showPlotterWindow);
           event.newGuest.setMenu(null);
           event.newGuest?.on('closed', () => {
-            electronWindow?.webContents.send('CLOSE_CHILD_WINDOW');
+            ipcMain.removeListener(SHOW_PLOTTER_WINDOW, showPlotterWindow);
+            electronWindow?.webContents.send(CLOSE_PLOTTER_WINDOW);
```

Comment:
Can you explain why it needs to be `electronWindow?.webContents` instead of `electronWindow.webContents`? The same applies to all `event.newGuest?.x`.

Reply:
Actually it's not needed, apart from here: https://github.com/arduino/arduino-ide/blob/2e3c5dee4bd1937731171a5b986110fa6eacce47/arduino-ide-extension/src/electron-main/theia/electron-main-application.ts#L337

I'm changing them, thanks!